.github/workflows/generate-js.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Generate JavaScript code and open pull request with it'
+
+on:
+  push:
```

Comment:
Could just do it against `main`, since we only really care if we're stale there.

Reply:
The main reason why I made this PR is to make and test changes in forks. So for example I would create a branch `new-feature` and make changes there, then the bot generates the code, I test it and then PR it. Other contributors can use it as well, just need to document it in CONTRIBUTING.md.